### ISSUE TITLE
Xcode 9 build with Ambassador SDK: iTC validation issue

### ISSUE THREAD

**Issue by siberianisaev** [GetAmbassador/ambassador-ios-pod#2]
Hi! I got following issue https://forums.developer.apple.com/thread/86290

```
ERROR ITMS-90022: "Missing required icon file. The bundle does not contain an app icon for iPhone / iPod Touch of exactly '120x120' pixels, in .png format for iOS versions >= 7.0."
		ERROR ITMS-90023: "Missing required icon file. The bundle does not contain an app icon for iPad of exactly '76x76' pixels, in .png format for iOS versions >= 7.0."
		ERROR ITMS-90023: "Missing required icon file. The bundle does not contain an app icon for iPad of exactly '167x167' pixels, in .png format for iOS versions supporting iPad Pro."
		ERROR ITMS-90023: "Missing required icon file. The bundle does not contain an app icon for iPad of exactly '152x152' pixels, in .png format for iOS versions >= 7.0."
```
See also https://github.com/CocoaPods/CocoaPods/issues/7003#issuecomment-330721443

**If I remove Ambassador pod, then all is ok with build validation.**

Think Ambassador->Resources->Ambassador.bundle->Assets.car overrides Application Bundle icons asset. I have tried to move Copy Bundle Resources build phase as recommended there https://github.com/CocoaPods/CocoaPods/issues/7003#issuecomment-330743187 but without result.

Coud it be fixed ASAP?

**Comment by jarell-lloyd** [comment]
@siberianisaev Are you still experiencing this issue? If so, what OS and Xcode version are you using?

**Comment by siberianisaev** [comment]
Yep. Xcode 9.0, OS 10.12.6

**Comment by siberianisaev** [comment]
The only workaround I found is to manually remove **Assets.car** from Ambassador.bundle

**Comment by jarell-lloyd** [comment]
@siberianisaev Here are some additional workarounds in this thread: https://stackoverflow.com/questions/46462871/missing-required-icon-files-in-build-from-xcode-9/46799223#46799223

**Comment by siberianisaev** [comment]
As I understand was fixed in v1.0.12
